VMEncryption/main/EncryptionSettingsUtil.py

```diff
@@ -55,8 +55,9 @@ def increment_index(self):
         # specify buffering = 0 and then use os.fsync to flush
         # https://docs.python.org/2/library/functions.html#open
         # https://linux.die.net/man/2/fsync
-        with open(CommonVariables.encryption_settings_counter_path, "w", 0) as outfile:
-            outfile.write(str(index + 1) + "\n")
+        with open(CommonVariables.encryption_settings_counter_path, "wb", 0) as outfile:
```

Comment:
wb

Reply:
changed to "w" with ascii encoding using python2 + python3 compatible syntax for io.open and builtin str